src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1932,6 +1932,17 @@ internal enum ErrorCode
 
         #endregion diagnostics introduced for C# 9.0
 
+        #region diagnostics introduced in C# 10.0
+        WRN_CallerArgumentExpressionParamForUnconsumedLocation = 8912,
+        ERR_NoConversionForCallerArgumentExpressionParam = 8913,
+        ERR_BadCallerArgumentExpressionParamWithoutDefaultValue = 8914,
+        WRN_CallerLineNumberPreferredOverCallerArgumentExpression = 8915,
+        WRN_CallerFilePathPreferredOverCallerArgumentExpression = 8916,
+        WRN_CallerMemberNamePreferredOverCallerArgumentExpression = 8917,
+        WRN_CallerArgumentExpressionAttributeHasInvalidParameterName = 8918,
```

Comment:
Should these have the default warning level (4), or a higher warning level (5 or 6)?

Reply:
They should be in line with the other caller argument warnings (ie, 0).

---
In reply to: [597425232](https://github.com/dotnet/roslyn/pull/51952#discussion_r597425232) [](ancestors = 597425232)